package.json

```diff
@@ -21,21 +21,27 @@
   "license": "MIT",
   "author": "Daniel Cousens",
   "files": [
-    "index.js"
+    "src"
```

Comment:
`ts_src` -> `src` and put compile output to `dist/` ?

Reply:
this way keeps the git history better ?

Reply:
No.